### PR TITLE
JDK-8259732: JDK 16 L10n resource file update - msg drop 10

### PR DESCRIPTION
**Pull request by leohacker**
This is the changes for JDK 16 msg drop 10.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8259732](https://bugs.openjdk.java.net/browse/JDK-8259732): JDK 16 L10n resource file update - msg drop 10

### Reviewers
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk16 pull/123/head:pull/123`
`$ git checkout pull/123`

### CODE REVIEW

**Comment by leohacker** [comment]
Since they're Unicode escape sequences in the l10n resource files, so I attached a human readable webrev, created by `git webrev` and converted. Pls find this to help your review. 
[webrev.tar.gz](https://github.com/openjdk/jdk16/files/5814846/webrev.tar.gz)

or review the changes from openjdk cr.
http://cr.openjdk.java.net/~ljiang/8259732/webrev.00/

**Comment by leohacker** [comment]
@naotoj Would you have time to take a look at this change? Very appreciated!

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties [hunk 1]
```diff
@@ -515,4 +515,4 @@ doclet.NoFrames_specified=\
 # L10N: do not localize the option name -footer
 doclet.footer_specified=\
     The -footer option is no longer supported and will be ignored.\n\
-    It may be removed in a future release.
\ No newline at end of file
+    It may be removed in a future release.
```
Comment (naotoj) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties:518]: I believe this is to fix no newline at the end (unrelated to l10n changes). Do we need to change the copyright year for this?

Comment (leohacker) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties:518]: Actually I was correcting the L217, changed {) -> {}. But 2 days ago, Jonathan Gibbons fixed it in another commit 6bb6093. I found his fix after running the merge. Looks both of us forgot to update the copyright year. Any suggestion? 
```
doclet.help.systemProperties.body=\
    The {0} page lists references to system properties.
```

Comment (naotoj) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties:518]: I believe your PR still contains the fix to add the newline at the end of the file (at L518). So I think you can simply change the copyright year in `standard.properties` file.

Comment (leohacker) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties:518]: Thx! Fixed the copyright year for this file.

**Review by naotoj — APPROVED** [review]

**Pull request closed without merging**